services/gcp/cloudkms/cryptoKeyVersions.yml

```diff
@@ -0,0 +1,77 @@
+name: Cloud KMS Crypto Key Versions
+description: >-
+  A key version contains key material used for encryption or signing.
+scope: CRITICAL
+notes: >-
+  Cloud KMS is an extremely sensitive service. Keys can be used for encryption/decryption of 
+  sensitive data or for the creation or verification of digital signatures.
+privileges:
+  create:
+    risks: [impact:spend]
+  destroy:
+    risks: [destruction:crypto, impact:encryption]
+    notes: >-
+      Destroyed key versions cannot be recovered.
+      Any data encrypted with the key version will no longer be able to be decrypted.
+  get:
+    risks: [discovery:infra]
+  list:
+    risks: [discovery:infra]
```

Comment:
```suggestion
  get:
    risks: [discovery:infra]
    notes: >-
      This includes infra discovery because key metadata such as the algorithm are exposed. Does not give access to keys.
  list:
    risks: [discovery:infra]
    notes: >-
      See get.
```